metadata.json

```diff
@@ -18,11 +18,11 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.4.0 < 2015.3.0"
     },
     {
       "name": "puppet",
-      "version_requirement": ">= 3.3.0 < 4.0.0"
+      "version_requirement": ">= 3.3.0 < 5.0.0"
```

Comment:
Here's the commit that changed this incorrectly - https://github.com/puppetlabs/puppetlabs-acl/commit/2d332e210cae42a09cd9f7e9c2f9c86c33c43ab6